### PR TITLE
GitHub Workflow to change install version

### PR DESCRIPTION
**Pull request by Alan-Cha**
Replace #981

### CODE REVIEW

File: mkdocs/mkdocs.yml [hunk 1]
```diff
@@ -185,9 +199,4 @@ nav:
     - Analytics: contributing/analytics.md
     - Tasks: contributing/tasks.md
   - Roadmap: roadmap.md
-  - News: news.md
-
-# Google Analytics
-google_analytics:
```
Comment (kalantar) [inline mkdocs/mkdocs.yml:191]: Did we intend to remove google_analytics directive?

File: .github/workflows/ind.yaml [hunk 2]
```diff
@@ -29,15 +24,11 @@ jobs:
...
         fi
-    - name: Get owner and repo
-      run: |
-        ownerrepo=${{ github.repository }}
-        owner=$(echo $ownerrepo | cut -f1 -d/)
-        if [[ "$owner" == "iter8-tools" ]]; then
-          owner=iter8
-        fi
-        echo "OWNER=$owner" >> $GITHUB_ENV
-        echo "REPO=ind" >> $GITHUB_ENV
```
Comment (kalantar) [inline .github/workflows/ind.yaml:40]: aren't OWNER and REPO still needed? I still see them in the `tags` argument to build-push-action

File: .github/workflows/change-install-version.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        elif [[ "${{ github.ref }}" == *"master" ]]; then
+          echo "VERSION=latest" >> $GITHUB_ENV
+        fi
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: pip3 install -r ./.github/workflows/requirements.txt
+    - name: Edit install version
+      run: python3 ./.github/workflows/change-install-version.py ${{ env.VERSION }}
```
Comment (kalantar) [inline .github/workflows/change-install-version.yaml:27]: I'm not understanding how this updates `mkdocs/docs/getting-started/install.md`. I see the py script updates only `mkdocs.yml`.

File: .github/workflows/change-install-version.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    - uses: EndBug/add-and-commit@v7
```
Comment (kalantar) [inline .github/workflows/change-install-version.yaml:28]: Do we want to just commit? Or do we want to create PR as well?

**Review by kalantar — COMMENTED** [review]
I have some questions...

File: mkdocs/mkdocs.yml (hunk 1 above)
Comment (Alan-Cha) [reply mkdocs/mkdocs.yml:191]: This method of adding Google Analytics is deprecated. I've added an updated version.

File: mkdocs/mkdocs.yml [hunk 5]
```diff
@@ -53,9 +53,20 @@ plugins:
...
+  - macros:
+      # Replace {{ variable }} syntax with [[ variable ]] to avoid conflicts
+      j2_block_start_string: '[[%'
+      j2_block_end_string: '%]]'
+      j2_variable_start_string: '[['
+      j2_variable_end_string: ']]'
         
 # Customization
 extra:
+  # Google Analytics
```
Comment (Alan-Cha) [inline mkdocs/mkdocs.yml:65]: @kalantar This is the new method of adding Google Analytics. You can read more about it [here](https://github.com/iter8-tools/iter8/pull/981#discussion_r693106545).

File: .github/workflows/change-install-version.yaml (hunk 3 above)
Comment (Alan-Cha) [reply .github/workflows/change-install-version.yaml:27]: I've replaced the install version with a variable. Now we can dynamically update the install version by modifying `mkdocs.yml`.

File: mkdocs/docs/getting-started/install.md [hunk 6]
```diff
@@ -9,7 +9,7 @@ hide:
 Install Iter8 in your Kubernetes cluster as follows. This step requires [Kustomize v3+](https://kubectl.docs.kubernetes.io/installation/kustomize/).
 
 ```shell
-export TAG=v0.7.19
+export TAG=[[ iter8.install_version ]]
```
Comment (Alan-Cha) [inline mkdocs/docs/getting-started/install.md:12]: @kalantar This is the install version variable.

File: mkdocs/mkdocs.yml [hunk 7]
```diff
@@ -64,7 +75,10 @@ extra:
       link: https://github.com/iter8-tools/iter8
     - icon: fontawesome/brands/slack
       link: https://join.slack.com/t/iter8-tools/shared_invite/zt-awl2se8i-L0pZCpuHntpPejxzLicbmw
-
+  
+  iter8:
+    install_version: v0.7.19
```
Comment (Alan-Cha) [inline mkdocs/mkdocs.yml:80]: @kalantar This is where we can change the install version in the `mkdocs.yml`.

File: .github/workflows/ind.yaml (hunk 2 above)
Comment (Alan-Cha) [reply .github/workflows/ind.yaml:40]: I think this may have been a mistake.

File: .github/workflows/change-install-version.yaml (hunk 4 above)
Comment (Alan-Cha) [reply .github/workflows/change-install-version.yaml:28]: I think committing is less abrasive. If we were to go the other route, then we would need to wait for a workflow and accept a PR after every new release. I think we go with commits for now and we can change it in the future if the requirements are different.

Comment (kalantar) [reply .github/workflows/change-install-version.yaml:28]: This isn't just commit; it is push as well. Any opinions @sriumcp and @huang195 ?

File: mkdocs/mkdocs.yml (hunk 7 above)
Comment (kalantar) [reply mkdocs/mkdocs.yml:80]: Need to be updated already.

**Comment by Alan-Cha** [comment]
See #995

**Pull request closed without merging**